cmd/system-probe/windows/service/service.go

```diff
@@ -39,6 +39,11 @@ func (m *sysprobeWindowService) Execute(args []string, r <-chan svc.ChangeReques
 	changes <- svc.Status{State: svc.StartPending}
 
 	if err := app.StartSystemProbe(); err != nil {
+		if err == ErrNotEnabled {
+			changes <- svc.Status{State: svc.Running, Accepts: cmdsAccepted}
+			return
```

Comment:
Do we still need to do `changes <- svc.Status{State: svc.Stopped}` before returning?

Reply:
I'm not sure – @derekwbrown would you know that?

Reply:
IDK what the right thing here is.

Reply:
I found a very helpful state diagram. Still reading to determine what is required here: https://docs.microsoft.com/en-us/windows/win32/services/service-status-transitions

Reply:
OK, It looks like we need to send `svc.Stopped`.

Reply:
Thanks! Updated